main.js

```diff
@@ -70,6 +72,11 @@ function saveOutfit() {
     <i class="far fa-times-circle"></i>
   </div>`)
   saveForm.reset();
+  removeChoice(allGarments);
+  resetGarmentSelection(allButtons);
+  enableButton();
```

Comment:
Since we are using this function to disable the submitButton I think we should rename the enableButton function to toggleButtonEnable or something thats a bit more descriptive lines 77 and 82.  I think it would also be good for cleaner code to put lines 74-78 into a helper function that is called like resetPage or something along those lines.